docs/source/configuring_zipkin.rst

```diff
@@ -104,8 +104,9 @@ fine tune as per your use case.
 
 5. zipkin.trace_id_generator
 ----------------------------
-    A method definition to generate a `trace_id` for the request. By default,
-    it creates a random trace id otherwise.
+    A method definition to generate a `trace_id` for the request. The trace_id is
+    expected to be a 64-bit hex string By default, it creates a random trace id
```

Comment:
Need a period between "hex string" and "By". I'd also like an example string, like "dfd0bda0f76a4c23", just to make sure people know it shouldn't have "0x" in front of it or anything.